examples/exotica_examples/scripts/example_ik_with_jnt_smoothing.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+import signal
+from pyexotica.publish_trajectory import publishPose, sigIntHandler
+from time import sleep
+import task_map_py
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64MultiArray as FloatArray
+
+DT = 1.0/100.0 # 100 HZ
+DAMP = 0.005
+
+SMOOTHING_TASK_MAPS = ['JointVel', 'JointAcc', 'JointJerk']
+
+XLIM = [0.0, 0.75]
+YLIM = [-0.5, 0.5]
+ZLIM = [0, 1.0]
+
+LIMITS = np.array([XLIM,\
+                   YLIM,\
+                   ZLIM])
+
+class Example(object):
+
+    def __init__(self):
+
+        # Set init variables
+        self.q=np.array([0.0]*7)
+        self.joy = None
+        self.eff = np.array([0.6, -0.1, 0.5, 0, 0, 0])
+
+        # Setup exotica
+        self.solver = exo.Setup.loadSolver('{exotica_examples}/resources/configs/ik_with_jnt_smoothing.xml')
+        self.problem = self.solver.getProblem()
+        self.task_maps = self.problem.getTaskMaps()
+        
+        # Setup ros subscriber
+        self.sub={}
+        self.sub['joy'] = rospy.Subscriber('/joy', Joy, self.callback)
+        
+    def callback(self, msg):
+        self.joy = msg
+
+    def update(self, event):
+        if self.joy is None: return
+
+        # Compute eff goal
+        dx = self.joy.axes[4]
+        dy = self.joy.axes[0]
+        dz = self.joy.axes[1]
+        eff = self.eff + DAMP * np.array([dx, dy, dz, 0, 0, 0])
+
+        # Check limits
+        for i in xrange(3):
+            if not LIMITS[i,0] <= eff[i] <= LIMITS[i,1]:
+                eff[i] = self.eff[i]
+
+        # Set eff goal 
+        self.problem.setGoal('Position', eff)
+        #self.problem.getScene().attachObjectLocal('Goal', '', exo.KDLFrame(eff))
+                
+        # Setup problem
+        if self.joy.buttons[0]:
+            # Button [0] pressed -> turn smoothing off -> report 
+            rho = 0
+            print "Smoothing OFF"
```

Comment:
For future compatibility, please use Python3-style printing (``print("Smoothing OFF")``) and at the top ``from __future__ import print_function``.